src/sdk/cpp/lib/tests/test_talent_test.cpp

```diff
@@ -0,0 +1,313 @@
+/*****************************************************************************
```

Comment:
test_talent.cpp to use the "test" pre-fix only?

Reply:
Since the file containing the code we're testing is called `talent_test.cpp` and the convention has been to name the test files `test_<filename>.cpp` it follow that the test file coresponding to `talent_test.cpp` should be named `test_talent_test.cpp`. It's not pretty I agree and I welcome suggestions for a better name to `talent_test.cpp`.

Reply:
What's about to rename talent_test.*pp to testset_talent.*pp or something like talent_test_helper?

Reply:
Done